client/js/renderPreview.js

```diff
@@ -61,3 +61,56 @@ function handleImageInPreview(content, container) {
 		});
 	}
 }
+
+/* Image viewer */
+
+// FIXME Remove #input focus when this is open
+$("#viewport").on("click", ".toggle-content img", function() {
+	const self = $(this);
+	const link = self.closest("a");
+
+	openImageViewer(self, link);
+
+	// Prevent the link to open a new page since we're opening the image viewer,
+	// but keep it a link to allow for Ctrl/Cmd+click
+	event.preventDefault();
+});
+
+// Reason to make this a specific handler is, for images, to open the image
+// viewer even when clicking in the margins of the thumbnail
+$("#viewport").on("click", ".toggle-content.toggle-type-image", function() {
```

Comment:
I'll rewrite that part to make it simpler and remove an edge case (accidentally clicking on margin of image in link previews, not image previews, will open the link instead of the viewer).